platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/transaction/Transaction.java

```diff
@@ -44,6 +45,7 @@ public sealed interface Transaction permits ConsensusTransaction {
      *
      * @return the application transaction Bytes or {@code Bytes.EMPTY} if the transaction is a system transaction
      */
+    @Deprecated
```

Comment:
I think we will continue to use this method in the future, so it should not be deprecated

Reply:
Updated